internal/constants/keys_enterprise.go

```diff
@@ -0,0 +1,16 @@
+//go:build enterprise
```

Comment:
nit: Should we have another build flag here? Maybe `release` or `!development`.
I can imagine having more `enterprise` features in the future, and the need to test them before a release. Which could lead to conflicts reusing the build flag.
I am also fine with keeping it, and changing it once it becomes necessary.